packages/itwinui-react/package.json

```
@@ -19,7 +19,12 @@
       }
     },
     "./react-table": {
-      "types": "./react-table.d.ts"
+      "import": {
+        "types": "./esm/react-table/react-table.d.ts"
+      },
+      "require": {
+        "types": "./cjs/react-table/react-table.d.ts"
+      }
```

Would this also do the same thing?

```suggestion
      "import": {
        "types": "./react-table.d.ts"
      },
      "require": {
        "types": "./react-table.d.ts"
      }
```

There was never meant to be a `react-table` folder inside `esm/` and `cjs/` folder, so it would be nice to not depend on it.

---

Just tried it out with your suggestion and it doesn't get picked up, I still get the error about unresolved path :(

---

Ok, lets go with this one for now:

```suggestion
      "types": "./react-table.d.ts",
      "import": {
        "types": "./esm/react-table/react-table.d.ts"
      },
      "require": {
        "types": "./cjs/react-table/react-table.d.ts"
      }
```